app/use_cases/generate_slack_message.js

```diff
@@ -1,18 +1,22 @@
+const moment = require("moment");
+
 class GenerateSlackMessage {
   constructor(options) {
     this.firstName = options.firstName;
   }
 
   execute({ lunchCycle }) {
-    const twentFourHoursInMs = 86400000;
-
-    let message = `Hey ${this.firstName} it’s time to enter the draw for the next cycle of company lunches. Let us know which dates you’ll be available on by reacting with the matching emoji.\n\n`;
+    if (this.firstName === null) {
+      var message = `Hey {first name}! it’s time to enter the draw for the next cycle of company lunches. Let us know which dates you’ll be available on by reacting with the matching emoji.\n\n`;
+    } else {
+      var message = `Hey ${this.firstName} it’s time to enter the draw for the next cycle of company lunches. Let us know which dates you’ll be available on by reacting with the matching emoji.\n\n`;
```

Comment:
I don't like the `if () { } else {}` here.

It's got difference in the `}! It's` and we have two basically identical strings.

Suggest you change it to something like:

```javascript 
let firstName = this.firstName;
if (firstName === null) {
  firstName = '{First Name}';
}

let message = `Hey ${firstName}! It’s...
```